spec/the_google/calendar_test.rb

```diff
@@ -201,6 +201,21 @@
                 .returns api_result
         end
 
+        describe 'and the events do not have start or end values' do
+
+          let(:items) do
+            [
+              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
+              Struct.new(:summary, :start, :end).new(random_string, nil, nil),
```

Comment:
I'm not going to do anything with this issue.  I'm only speaking up to try to help avoid what might happen further downstream to any client.  Perhaps nothing, but I bet it would will either throw in a worker or it would alter the availability calculations.

I think this only exists as my project because we were short on github private repos and I was concerned about open eyes on active development.  I'm not concerned any longer, and if there are open slots in GH I recommend hooking any clients to a private fork.  If this happens, email me and I'll delete this repo.  Otherwise I'll keep it here as-is so current builds are not broken.